Formula/convox.rb

```diff
@@ -19,21 +20,27 @@ class Convox < Formula
   end
 
   def install
-    ENV["GOPATH"] = buildpath
+    ENV["GOPATH"] = buildpath/"go"
 
-    (buildpath/"src/github.com/convox/rack").install Dir["*"]
+    (buildpath/"src").install Dir["*"]
 
     resource("packr").stage { system "go", "install", "./packr" }
-    cd buildpath/"src/github.com/convox/rack" do
-      system buildpath/"bin/packr"
+
+    cd buildpath/"src" do
+      system "../go/bin/packr"
+      system "go", "build", "-mod=vendor", "-ldflags=-X main.version=#{version}",
+             "-o", bin/"convox", "-v", "./cmd/convox"
     end
 
-    system "go", "build", "-ldflags=-X main.version=#{version}",
-           "-o", bin/"convox", "-v", "github.com/convox/rack/cmd/convox"
     prefix.install_metafiles
   end
 
   test do
-    system bin/"convox"
+    require "open3"
```

Comment:
This is not needed, there's a homebrew dsl to read the shell output. `shell_output "<cmd>", <exitcode>`

Reply:
shell_output did not appear to capture stderr

Reply:
I was going off of the information found here: https://www.rubydoc.info/github/Homebrew/brew/Formula.test

Reply:
> shell_output did not appear to capture stderr

Not by default but you can just use `cmd 2>&1` to do that.

Reply:
The test seems more accurate as is

Reply:
The test also depends on gems that not everyone might have installed and is out of line with homebrew standards like this. Just use the homebrew DSL, it's there for a reason.

Reply:
Open3 is part of the Ruby standard library

https://www.rubydoc.info/stdlib/open3/Open3.popen3

Reply:
That doesn't change the homebrew standards. It's not really up for debate, it's not getting merged like this.